### ISSUE TITLE
[18.0] module: description

### ISSUE THREAD

**Issue by QaidjoharBarbhaya** [odoo/odoo#186406]
### Odoo Version

- [X] 16.0
- [X] 17.0
- [X] 18.0
- [ ] Other (specify)

### Steps to Reproduce

- Install l10n_pe_edi module
- Create invoice with document type with code 08 Nota de Debito

Current Behaviour: Error from OSE as shown in screen recording: https://youtu.be/n4ZMvvfhL9g
Expected Behaviour: Direct debit notes with no original invoice should register with OSE successfully.

**Comment by QaidjoharBarbhaya** [comment]
@william-andre Hello sir, can you please confirm the legitimacy of this issue?

**Comment by william-andre** [comment]
@smetl maybe?

**Comment by QaidjoharBarbhaya** [comment]
@smetl Hello sir, can you please look into this? Thank you.

**Comment by smetl** [comment]
@QaidjoharBarbhaya Hello. Sorry for the delay, I was on vacation. I'll have a look into this

**Comment by fvz-odoo** [comment]
Hello @QaidjoharBarbhaya 

Based on the government website https://www.gob.pe/26401-nota-de-debito a debit note is a document to modify an invoice or a boleta. If you want to create a debit note, be sure to make it from a original invoice (or boleta) to have the reference. If you think this is a mistake, could you please share the legal reference? 

Thanks!

Fernando

@smetl @pmax-x

**Comment by avollmers** [comment]
Hello @fvz-odoo 
According to the law that governs invoices (https://cdn.www.gob.pe/uploads/document/file/3052447/Reglamento%20de%20Comprobantes%20de%20Pago.pdf?v=1651242244), in Article 10, numeral 2.1 literal a) the second paragraph states that, exceptionally, the customer may issue a Debit Note to the Supplier, in which case there is no invoice to apply it to. It is a rare case, but some users of the Peruvian localization are confused that Odoo does have the option to issue a Debit Note without an invoice, but then reports an error when the user does so.

**Comment by desdelinux** [comment]
> Hello  Hola@fvz-odoo According to the law that governs invoices (https://cdn.www.gob.pe/uploads/document/file/3052447/Reglamento%20de%20Comprobantes%20de%20Pago.pdf?v=1651242244), in Article 10, numeral 2.1 literal a) the second paragraph states that, exceptionally, the customer may issue a Debit Note to the Supplier, in which case there is no invoice to apply it to. It is a rare case, but some users of the Peruvian localization are confused that Odoo does have the option to issue a Debit Note without an invoice, but then reports an error when the user does so.De acuerdo a la ley que rige las facturas (https://cdn.www.gob.pe/uploads/document/file/3052447/Reglamento%20de%20Comprobantes%20de%20Pago.pdf?v=1651242244), en el artículo 10, numeral 2.1 literal a) el segundo párrafo señala que, excepcionalmente, el cliente podrá emitir una Nota de Débito al Proveedor, en cuyo caso no existe factura a la cual aplicarla. Es un caso raro, pero algunos usuarios de la localización peruana están confundidos de que Odoo tenga la opción de emitir una Nota de Débito sin factura, pero luego informe un error cuando el usuario lo hace.

Debit Notes issued from "los sistemas del contribuyente" must comply with SUNAT regulations, as detailed here:

[Debit Note - SUNAT](https://cpe.sunat.gob.pe/tipos_de_comprobantes/nota_de_debito?utm_source=chatgpt.com)

The key point from this document is:

_"Puede modificar “una” FE (factura electrónica) que cuente con CDR “aceptada” otorgada o BVE (boleta de venta electrónica) otorgada con anterioridad."_

Additionally, in the XML - UBL 2.1 schema, which is mandatory for electronic Debit Notes according to SUNAT, the node /DebitNote/cac:BillingReference/cac:DebitNoteDocumentReference is required to directly reference the original document being modified.

In summary, it is not possible to issue Debit Notes without referencing an invoice (Factura, Boleta), as this is outside the scope and even the SUNAT portal. This is not a bug in the system but a regulatory restriction.

**Comment by avollmers** [comment]
This is the validation response when issuing from a test PSE account:

"Solo puede usar SIN DOCUMENTO A MODIFICAR si el Tipo de Nota es Penalidades / Otros Conceptos | Error al PreValidar INFO : 2524 (nodo: "cac:DiscrepancyResponse/cbc:ResponseCode" valor: "01")"

Which confirms regulation above and Sunat precedent.

However, if this will not pass OSE validation, should we then remove the (08) options from the domain like (07) are currently in order to prevent the user from creating this type of document from this view?

**Comment by fvz-odoo** [comment]
Hello @avollmers and @QaidjoharBarbhaya I will close this PR as this is not necessary. I will keep the feedback of removing from the domain the type of document for better usability, although we have other bigger priorities as of now. Cheers!

Fernando

@smetl @william-andre @pmax-x @desdelinux
